tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -395,10 +395,15 @@ public async Task ConcurrentJobConfigurationShouldBeRespected()
         runningJobs.Count.ShouldBe(2);
         runningJobs[0].CorrelationId.ShouldNotBe(runningJobs[1].CorrelationId);
 
-        static bool AtLeastTwoJobsAreInitializing(IList<ExecutionProgress> events)
+        static ExecutionProgress? AtLeastTwoJobsAreInitializing(IList<ExecutionProgress> events)
```

Comment:
The method name doesn't really fit anymore.

But I am also not sure what is a good name that isn't 70 characters long

Reply:
Updated with something less than 70 chars long and a (little) bit more expressive.

Better?

Reply:
FWIW, I'm going to go over this whole test suite as part of a follow up PR to migrate it to event based.

Reply:
How about `LatestOfAtLEastTwoInitializingJobs`?

Reply:
Fixed